src/base/static/components/organisms/map-layer-group.js

```diff
@@ -0,0 +1,131 @@
+import React from "react";
```

Comment:
nit: I think MapLayerGroup should be a molecule: http://atomicdesign.bradfrost.com/chapter-2/#molecules